packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.scss

```diff
@@ -4,40 +4,43 @@
 *--------------------------------------------------------------------------------------------*/
 
 .presentation-instance-filter-dialog {
-  min-width: 50vw;
+  .presentation-instance-filter-dialog-content-container {
+    min-height: calc(var(--iui-size-2xl) * 5 + var( --iui-size-xs));
```

Comment:
I think `min-height` can be simplified to `calc(var(--iui-size-2xl) * 6)`.

Reply:
No, the instance filter dialog would be too big then.

Reply:
changed to other calculation